InMobiAdapter/src/main/java/com/chartboost/mediation/inmobiadapter/InMobiAdapter.kt

```diff
@@ -122,18 +124,25 @@ class InMobiAdapter : PartnerAdapter {
                 val gdprConsent = gdprApplies?.let { buildGdprJsonObject(it) }
 
                 return suspendCoroutine { continuation ->
-                    InMobiSdk.init(context.applicationContext, accountId, gdprConsent) { error ->
-                        continuation.resume(
-                            error?.let {
-                                PartnerLogController.log(SETUP_FAILED, "${it.message}")
-                                Result.failure(ChartboostMediationAdException(ChartboostMediationError.CM_INITIALIZATION_FAILURE_UNKNOWN))
-                            } ?: run {
-                                Result.success(
-                                    PartnerLogController.log(SETUP_SUCCEEDED)
+                    InMobiSdk.init(
+                        context = context.applicationContext,
+                        accountId = accountId,
+                        consentObject = gdprConsent,
+                        sdkInitializationListener = object : SdkInitializationListener {
+                            override fun onInitializationComplete(error: Error?) {
+                                continuation.resume(
+                                    error?.let {
+                                        PartnerLogController.log(SETUP_FAILED, "${it.message}")
+                                        Result.failure(ChartboostMediationAdException(ChartboostMediationError.CM_INITIALIZATION_FAILURE_UNKNOWN))
+                                    } ?: run {
+                                        Result.success(
+                                            PartnerLogController.log(SETUP_SUCCEEDED)
+                                        )
+                                    }
                                 )
                             }
-                        )
-                    }
```

Comment:
I guess it's because `sdkInitializationListener` went from an `@Nullable SdkInitializationListener` to `SdkInitializationListener?`. Not sure why that would break it though